Makefile

```diff
@@ -0,0 +1,24 @@
+SHELL := /bin/bash
+
+start-deps:
+	docker-compose -f test/docker-compose.yml up -d
```

Comment:
Since these are mainly used for development purposes maybe we should add `AcceptEULA=yes docker-compose...`